### ISSUE TITLE
[studio-ui] Typo in Site Dashboard publishing message

### ISSUE THREAD

**Issue by AndyReiter** [craftercms/craftercms#5922]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] The issue is in the latest released 4.0.x
- [ ] The issue is in the latest released 3.1.x

### Describe the issue

There is a typo in the site dashboard publishing message after publishing more than 1 item. The message says "The selected items has been published to live. Them will be visible soon" which is incorrect.

### Steps to reproduce

Steps:

1. Create a site based on editorial
2. Publish the home page in order to trigger a publish of the entire site
3. Edit two pages in order for them to reflect as recently edited items in the site dashboard
4. Navigate to the site dashboard and scroll down to the "Recent Activity" section
5. Select the two pages and click publish
6. Notice the message in the bottom left and the grammar issues in it. The message should be "The selected items have been published to live. They will be visible soon".

### Relevant log output

```shell
N/A
```

### Screenshots and/or videos

![image](https://user-images.githubusercontent.com/42217570/216354915-f7078da3-354e-4d7b-9a8a-f1c40e3a9408.png)

**Comment by sumerjabri** [comment]
It seems to be in these files:
```env/i18n-legacy.ts:      '{count, plural, one {The selected item has been pushed to {environment}. It will be visible shortly} other {The selected items has been pushed to {environment}. Them will be visible shortly}}'
translations/ko.json:  "item.publishedNow": "{count, plural, one {The selected item has been pushed to {environment}. It will be visible shortly} other {The selected items has been pushed to {environment}. Them will be visible shortly}}",
translations/de.json:  "item.publishedNow": "{count, plural, one {The selected item has been pushed to {environment}. It will be visible shortly} other {The selected items has been pushed to {environment}. Them will be visible shortly}}",
translations/es.json:  "item.publishedNow": "{count, plural, one {The selected item has been pushed to {environment}. It will be visible shortly} other {The selected items has been pushed to {environment}. Them will be visible shortly}}",```

**Comment by AndyReiter** [comment]
Confirmed fixed in latest build of 4.0-SNAPSHOT.
